ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsTimestamps.scala

```diff
@@ -15,6 +15,11 @@ import scala.util.Success
 private[backend] trait StorageBackendTestsTimestamps extends Matchers with StorageBackendSpec {
   this: AsyncFlatSpec =>
 
+  private val parameterStorageBackend: ParameterStorageBackend =
+    backendFactory.parameterStorageBackend
```

Comment:
Right now each test trait creates its own storage backends. Can we create these instances in `StorageBackendProvider` (and perhaps automatically reset them between each test)? Or should each individual test create its own instances?

Reply:
I will refactor testing code as the state lands. Good idea